module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -901,4 +901,18 @@ protected function isLocalUrl(string $url): bool
         $baseUrlNorm = $this->normalizeUrlForComparison($this->getServerUrl('home'));
         return str_starts_with($this->normalizeUrlForComparison($url), $baseUrlNorm);
     }
+
+    /**
```

Comment:
I've opened #3843 to move this improvement to a separate trait and apply it to the entire codebase. Once that gets merged, we'll want to remove the changes from here.

Reply:
I've just merged #3843, so you should be able to remove this function the next time you merge the dev branch into this one.